service/src/main/kotlin/no/nav/su/se/bakover/service/søknad/lukk/LukkSøknadServiceImpl.kt

```diff
@@ -194,7 +204,9 @@ internal class LukkSøknadServiceImpl(
         søknad: Søknad.Journalført.MedOppgave.IkkeLukket,
         lukketSøknadsbehandling: LukketSøknadsbehandling?,
     ): Either<KunneIkkeLukkeSøknad, Søknad.Journalført.MedOppgave.Lukket> {
-        val dokument = lagBrevRequest(person, søknad, request)
+        val sak = sakService.hentSak(fnr = person.ident.fnr)
```

Comment:
Jeg synes dette er litt dypt til å hente saken på nytt. Kan bytte ut hentSøknad på linje 58 med denne og sende med saksnummeret istedet. Man kan jo også argumentere for at det gir mening å legge på saksnummer i Søknad.kt

Reply:
Enig, men nå byttet jeg til hentSakIdOgSaksnummer som er mye lettere